magento/magento2
# Extract images from content only when new media gallery enabled

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1750: The content modules does not aware of Media Gallery status

### Manual testing scenarios (*)
1. Magento 2.4-dvelop installed
2. New media Gallery Disabled by default.
3. Navigate to Cms grid or Product Grid
4. Create new (product or page or block)
5. Attach image to the descriptions field of entity
6. Save entity (note that new media gallery disabled)

### Expected result (*)
Exception.log file is empty there is no any exception


### Questions or comments


### Contribution checklist (*)
